Python/ExcelOutputClass.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+Spyder Editor
+
+This is a temporary script file.
+"""
+
+import os
+import xlsxwriter
+import openpyxl
+
+os.chdir(r'\Users\Ben\Documents\GitHub\Ben-Projects')
+
+def make_excel_file(title):
```

Comment:
That didn't format nicely...
![image](https://cloud.githubusercontent.com/assets/9099993/23805657/07afe062-057c-11e7-9ccb-f996b2648d0f.png)